src/vendors/ethers/utils/gas-limit.ts

```diff
@@ -1,10 +1,10 @@
 import { ethers } from 'ethers';
 
 /**
- * This value has been calculated as the difference between estimated gas with caching on a single block
- * and actual gas without caching on multiple blocks.
+ * This value has been determined after testing all `order - fill` pairings and recording the difference
+ * between `ethers`' `estimateGas` and the tx's `gasUsed`.
  */
-const GAS_LIMIT_DELTA = ethers.BigNumber.from('15000');
+const GAS_LIMIT_DELTA = ethers.BigNumber.from('26000');
```

Comment:
i'd just put it at 30k, but i won't object.

Reply:
yeah, i was thinking about that, but that would make the gas price shown in MetaMask appear even higher (so not great from a UX perspective)... and as we had some solid tests that had a consistent upper limit of `25971`, i went with the 26k for now, also tested that, and it worked.

Reply:
Fine either way, with this merged I'll upgrade my node and test out more order variations